pkg/testutils/pgtest/datadriven.go

```diff
@@ -141,7 +141,7 @@ func RunTest(t *testing.T, path, addr, user string) {
 	}
 }
 
-func parseMessages(s string) []pgproto3.BackendMessage {
+func ParseMessages(s string) []pgproto3.BackendMessage {
```

Comment:
you will also need to add comments to these functions now that they are exported (in order to pass linting checks)

```
// ParseMessages parses a string containing multiple pgproto3 messages separated
// by the newline symbol. See testdata for examples ("until" or "receive"
// commands).
```

```
// MsgsToJSONWithIgnore converts the pgproto3 messages to JSON format. The
// second argument can specify how to adjust the messages (e.g. to make them
// more deterministic) if needed, see testdata for examples.
```